### ISSUE TITLE
如何在模型中添加regularization loss?添加后如何在训练时获取该loss?

### ISSUE THREAD

**Issue by wenston2006** [PaddlePaddle/models#3265]
请问regularization loss是否可通过某些optimizer中的regularization参数添加(如:class paddle.fluid.optimizer.LarsMomentumOptimizer(learning_rate, momentum, lars_coeff=0.001, lars_weight_decay=0.0005, regularization=None, name=None))? 此外如何获取训练中的regularization loss?

**Comment by wanghaoshuang** [comment]
可以参考这个文档：https://www.paddlepaddle.org.cn/documentation/docs/zh/1.5/api_cn/regularizer_cn.html#fluid-regularizer

但是，目前没有方法直接获取regularization loss

**Comment by wenston2006** [comment]
@wanghaoshuang 谢谢,我来看看
